Source/Lib/Codec/EbEncHandle.c

```diff
@@ -2997,15 +2997,20 @@ static EB_ERRORTYPE VerifySettings(\
         SVT_LOG("SVT [Error]: Instance %u: The constrained intra must be [0 - 1] \n", channelNumber + 1);
         return_error = EB_ErrorBadParameter;
     }
-   if (config->rateControlMode > 2) {
+    if (config->rateControlMode > 2) {
         SVT_LOG("SVT [Error]: Instance %u: The rate control mode must be [0 - 2] \n", channelNumber + 1); 
         return_error = EB_ErrorBadParameter;
     }
-	if ((config->rateControlMode == 2) && (config->crf > 51))
-	{
-		SVT_LOG("SVT [Error]: Instance %u:The crf value must be [0-51] \n", channelNumber + 1);
-		return_error = EB_ErrorBadParameter;
-	}
+    if (((config->rateControlMode == 0) || (config->rateControlMode == 1)) && (config->crf != 28))
+    {
```

Comment:
Hi @Austin-Hu , for multi-line, we prefer
if() {
}

Reply:
Changed it, thanks!